gradio/data_classes.py

```diff
@@ -110,6 +110,24 @@ class PredictBody(BaseModel):
         None  # dictionary of request headers, query parameters, url, etc. (used to to pass in request for queuing)
     )
 
+    def __get_pydantic_json_schema__(self, source_type: Any, handler):
```

Comment:
Based on the [docs](https://docs.pydantic.dev/latest/concepts/json_schema/#implementing-__get_pydantic_core_schema__), this should be a class method!

Reply:
Thank you, @freddyaboulton, for pointing that out! I've updated the method to be a class method as per the documentation. Please review the latest commit.

Reply:
Thanks - added a unit test